rpc/commands/lib.proto

```diff
@@ -42,6 +42,8 @@ message LibraryInstallReq {
     string name = 2;
     // The version of the library to install.
     string version = 3;
+    // Set to true to skip installation of specified library's dependencies.
+    bool noDeps = 4;
```

Comment:
What is the default? Can you please document it? Thank you!

Reply:
Updated, it's `false` by the way.